### ISSUE TITLE
App stops loading

### ISSUE THREAD

**Issue by timsueberkrueb** [tim-sueberkrueb/ubuntu-hangups#4]
http://paste.ubuntu.com/12252595/

```
qml: python loaded
Authenticating with refresh token
UbuntuWindow::handleSurfaceFocusChange(focused=false)
Authentication successful
Loading ...
Creating client ...
Adding client observer
UbuntuWindow::handleSurfaceFocusChange(focused=true)
UbuntuClipboard - Got invalid serialized mime data. Ignoring it.
UbuntuWindow::handleSurfaceFocusChange(focused=false)
libust[6577/6603]: Error: Error opening shm /lttng-ust-wait-5-32011 (in get_wait_shm() at lttng-ust-comm.c:958)
libust[6577/6602]: Error: Error opening shm /lttng-ust-wait-5 (in get_wait_shm() at lttng-ust-comm.c:958)
QObject::killTimer: Timers cannot be stopped from another thread
QObject::startTimer: Timers cannot be started from another thread
```

Thanks to Martin Kozub for reporting. Occurred at least on Nexus 7 LTE (ubuntu-rtm/devel) and Meizu MX4 (rc-proposed).

**Comment by zubozrout** [comment]
I've just noticed it downloaded some thumbnails to ~/.local/share/ubuntu-hangups.timsueberkrueb/cache/images/ that I sent recently and that there are "refresh_token.txt" and "settings.json" files two folders up. So I suppose my account works fine.﻿

**Comment by timsueberkrueb** [comment]
@zubozrout Okay, thanks for the info. Can you try the following for me: Open settings.json, set
`"cache_images": false` and try to start again?

**Comment by zubozrout** [comment]
@tim-sueberkrueb Hmm ... you know ... that just did the trick, it works now :+1: :)
Thanks for the workaround tip :)

**Comment by timsueberkrueb** [comment]
Fixed in patch 0.1.6.
